content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -10,13 +10,15 @@ weight: 50
 
 
 <!-- overview -->
+Taints are a Kubernetes node property that enable nodes to repel certain pods. Tolerations are a Kubernetes pod property that allow a pod to be scheduled on a node with a matching taint.
+
 [_Node affinity_](/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)
 is a property of {{< glossary_tooltip text="Pods" term_id="pod" >}} that *attracts* them to
 a set of {{< glossary_tooltip text="nodes" term_id="node" >}} (either as a preference or a
-hard requirement). _Taints_ are the opposite -- they allow a node to repel a set of pods.
+hard requirement). _Taints_ are the opposite and allow a node to repel a set of pods.
 
 _Tolerations_ are applied to pods. Tolerations allow the scheduler to schedule pods with matching
-taints. Tolerations allow scheduling but don't guarantee scheduling: the scheduler also
+taints. Tolerations allow scheduling but don't guarantee scheduling. The scheduler also
```

Comment:
This change isn't required and the text is correct as-is.

Reply:
@divya-mohan0209 
pointed changed has been reverted as per your instruction. 
Please confirm other changes.